packages/bot-web-ui/src/stores/transactions-store.ts

```diff
@@ -68,11 +68,38 @@ export default class TransactionsStore {
         );
         const statistics = trxs.reduce(
             (stats, { data }) => {
-                const { profit = 0, is_completed = false, payout = 0, buy_price = 0 } = data as TContractInfo;
+                const {
+                    profit = 0,
+                    is_completed = false,
+                    payout = 0,
+                    buy_price = 0,
+                    bid_price = 0,
+                } = data as TContractInfo;
                 if (is_completed) {
                     if (profit > 0) {
                         stats.won_contracts += 1;
                         stats.total_payout += payout;
```

Comment:
@maryia-matskevich-deriv, I don't think we need a loop for this since the payout and bid price are the same. If we don't receive a payout, can we use the bid price instead? Something like this: 
`stats.total_payout += payout ?? bid_price ?? 0`

Reply:
@rupato-deriv 
only for **accumulators** issue happens because `profit` comes always = `-1`
only for **multiplier** it happens because `profit` comes always = `0`

so we will never come here

at the same time for **accumulators** and for **multiplier** we get `payout` and `bid_price` values > 0
sometimes `bid_price` and `payout` have the same value but we need to add only one of them